app/models/node-storage.ts

```diff
@@ -5,6 +5,61 @@ import OsfModel from './osf-model';
 
 const { attr } = DS;
 
+export const TestCases: Record<'public' | 'private', Record<StorageStatus, { warning: boolean, error: boolean }>> = {
```

Comment:
@futa-ikeda I don't think this is a good place for this const. It's not the model's concern and negatively affects readability. You could instead create a module that exports this const under `tests/fixtures/`